spec/models/assignment_spec.rb

```diff
@@ -65,6 +67,37 @@
     it 'should require case sensitive unique value for short_identifier' do
       assignment = create(:assignment)
       expect(assignment).to validate_uniqueness_of(:short_identifier)
+      end
+    it 'should have a nil parent_assignment by default' do
+      assignment = create(:assignment)
+      expect(assignment.parent_assignment).to be_nil
+    end
+    it 'should have a nil peer_review by default' do
+      assignment = create(:assignment)
+      expect(assignment.peer_review).to be_nil
+    end
+    it 'should not be a peer review if there is no parent_assignment_id' do
+      assignment = create(:assignment)
+      expect(assignment.parent_assignment_id).to be_nil
+      expect(assignment.is_peer_review?).to be false
+    end
+    it 'should be a peer review if it has a parent_assignement_id' do
+      parent_assignment = create(:assignment)
+      assignment = create(:assignment, parent_assignment: parent_assignment)
+      expect(assignment.is_peer_review?).to be true
+      expect(parent_assignment.is_peer_review?).to be false
+    end
+    it 'should give a true has_peer_review_assignment result if it does' do
+      parent_assignment = create(:assignment)
+      assignment = create(:assignment, parent_assignment: parent_assignment)
+      expect(parent_assignment.has_peer_review_assignment?).to be true
+      expect(assignment.has_peer_review_assignment?).to be false
+    end
+    it 'should find children assignments when they reference the parent' do
+      parent_assignment = create(:assignment, id: 1)
```

Comment:
Don't hard code the ids in test objects (it can mess up some later validations, even though it doesn't in this case).